src/duplo_resource/tenant.py

```diff
@@ -545,3 +593,64 @@ def dns_config(self,
     tenant_id = tenant["TenantId"]
     response = self.duplo.get(f"v3/subscriptions/{tenant_id}/aws/dnsConfig")
     return response.json()
+
+  @Command()
+  def add_user(self, 
+    name: args.NAME) -> dict:
+    """Add User to Tenant
+    
+    Usage: CLI Usage
+      ```sh
+      duploctl tenant add_user <user> --tenant <tenant_name>
+      ```
+
+    Args:
+      name: The name of the user to add to the tenant.
+      tenant: The name of the tenant to add the user to.
+
+    Returns:
+      message: A message indicating the user was added to the tenant.
+    """
+    tenant_id = self.find(self.duplo.tenant)["TenantId"]
```

Comment:
The find method already checks for `self.duplo.tenant` so you only need to do this `self.find()`